tools/rosout/rosout.cpp

```diff
@@ -112,9 +114,10 @@ class Rosout
 
     rosout_sub_ = node_.subscribe("/rosout", 0, &Rosout::rosoutCallback, this);
     std::cout << "subscribed to /rosout" << std::endl;
+
```

Comment:
Please avoid unrelated whitespace changes.

Same a few lines below.

Reply:
removed the added whitespace at line 117 - sorry! I didn't see any other whitespace changes, which line do you mean?